app/javascript/dashboard/components/layout/AvailabilityStatus.vue

```diff
@@ -18,6 +18,23 @@
       </woot-button>
     </woot-dropdown-item>
     <woot-dropdown-divider />
+    <woot-dropdown-item class="auto-offline--toggle">
+      <woot-switch
+        size="small"
+        class="auto-offline--switch"
+        :value="currentUserAutoOffline"
+        @input="updateAutoOffline"
+      />
+      <span class="auto-offline--text">
```

Comment:
<img width="464" alt="Screenshot2022-12-16 at 12 54 18@2x" src="https://user-images.githubusercontent.com/15716057/208045325-4a808994-87f5-41d1-8f38-fe700b77b974.png">

This feels a little more natural